puppet/modules/consular/manifests/init.pp

```diff
@@ -1,36 +1,17 @@
 class consular(
   $consular_args = [],
 ) {
-  class { 'python' :
-    version    => 'system',
-    pip        => 'present',
-    dev        => 'present',
-    virtualenv => 'present',
-    gunicorn   => 'absent',
-  }
-
-  package { [
-    'build-essential',
-    'libssl-dev',
-    'libffi-dev',
-  ]: }
+  apt::ppa { 'ppa:jerith/consular': ensure => 'present' }
   ->
-  file { '/var/consular':
-    ensure => directory,
-  }
-  ~>
-  file { '/var/consular/requirements.txt':
-    content => 'consular',
-  }
-  ~>
-  python::virtualenv { '/var/consular/venv':
-    requirements => '/var/consular/requirements.txt',
-  }
-  ~>
   file { '/etc/init/consular.conf':
     content => template('consular/init.consular.conf.erb'),
   }
   ~>
+  package { 'python-consular':
```

Comment:
[Here](https://launchpad.net/~jerith/+archive/ubuntu/consular) it looks like the package is just called `consular`?